### ISSUE TITLE
How does the output logits value judge whether it is real or fake?

### ISSUE THREAD

**Issue by tapas** [ahaliassos/LipForensics#13]
To predict as real or fake from logits, can I sigmoid(logits) and put some threshold say 0.3 to classify as real or fake ? Or shall I directly threshold logits ? If yes can you suggest some threshold ? Thanks
